### PR TITLE
Add default location for mapper indices

### PR DESCRIPTION
**Pull request by sven1103**
and fail workflow, if path is not provided

Solves #37

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -24,6 +24,7 @@ params {
   enumerations = 1
   beta = 0.009
   prefix = 'hla_run'
+  index = 'data/indices/yara/hla_reference_dna'
```
[the remark this answers is not in the document]
Comment (sven1103) [reply nextflow.config]: you mean `"$baseDir/assets/multiqc_config.yaml"`, right?

[the remark this answers is not in the document]
Comment (apeltzer) [reply nextflow.config]: Ahem not the multiqc config, but I copied this over from rnaseq :-)

[the remark this answers is not in the document]
Comment (apeltzer) [reply nextflow.config]: `"$baseDir/data/indices/yara/hla_reference_dna"` would be correct

[the remark this answers is not in the document]
Comment (sven1103) [reply nextflow.config]: yeah, i meant your path was wrongly formatted, not the actual path ;)

**Pull request merged**